iceoryx_posh/test/moduletests/test_gw_gateway_discovery.cpp

```diff
@@ -14,22 +15,25 @@
 
 #include "test.hpp"
 
-#define private public
-#define protected public
```

Comment:
only 4 of the old sins left :)

Reply:
One of the hereditary sins of iceoryx...

Reply:
I added a task to the issue with the Windows port. I think two of them are quite trivial to fix, the other two might need some time.